### ISSUE TITLE
0.8.5 high severity vuln

### ISSUE THREAD

**Issue by asbrum** [shelljs/shelljs#1104]
### Node version (or tell us if you're using electron or some other framework):
v. 16.14.0
### ShellJS version (the most recent version/Github branch you see the bug on):
0.8.4, 0.8.5
### Operating system:
windows
### Description of the bug:
npm package usage blocked by vulnerability provider Sonatype OSS Index

https://ossindex.sonatype.org/vulnerability/sonatype-2014-0038?component-type=npm&component-name=shelljs&utm_source=proget&utm_medium=integration&utm_content=22.0.9.2

![sonatype_shelljs](https://user-images.githubusercontent.com/58296725/192774394-a1947294-12c9-4cab-811f-e25d3b930a30.png)

**Comment by nfischer** [comment]
Hey, thanks for notifying me about this. That security report on sonatype is a false positive and this is **not** a security vulnerability in ShellJS. Please see the statement I made [here](https://github.com/shelljs/shelljs/pull/524#issue-181857605) and [here](https://github.com/shelljs/shelljs/issues/945#issuecomment-506019725).

`shell.exec()` is **designed** to allow arbitrary command execution. It is the responsibility of the caller to use `shell.exec()` responsibly and to sanitize user inputs.

**Comment by nfischer** [comment]
Duplicate of issue #945.
